src/mono/wasm/debugger/DebuggerTestSuite/MiscTests.cs

```diff
@@ -928,5 +928,15 @@ public async Task InspectLocalsUsingClassFromLibraryUsingDebugTypeFull()
             );
         }
         //TODO add tests covering basic stepping behavior as step in/out/over
+
+        [Fact]
+        public async Task SetBreakpointInProjectWithSpecialCharactersInPath()
+        {
+            var bp = await SetBreakpointInMethod("debugger-test-special-char-in-path.dll", "DebuggerTests.CheckSpecialCharactersInPath", "Evaluate", 1);
+            Assert.True(
+                bp.Value["breakpointId"].Value<string>().EndsWith("debugger-test-special-char-in-path-%23%40/test.cs"), 
+                "Failed to set a breakpoint in project with special characters in path."
+                );
```

Comment:
This should try hitting that breakpoint too.

Reply:
The error was originally visible already while setting but good point, added.